samples/bot-types-of-cards/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-types-of-cards-nodejs",
+    "source": "officeDev",
+    "title": "Types of Cards",
+    "shortDescription": "Demonstrating the feature where user can send types of cards using bot.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-types-of-cards/nodejs",
+    "longDescription": [
+      "Demonstrating the feature where user can send types of cards using bot."
+    ],
+    "creationDateTime": "2023-01-13",
+    "updateDateTime": "2023-01-13",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "javascript"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "express"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "bot"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "Image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/bot-types-of-cards/nodejs/Images/TypesOfCards.gif",
```

Comment:
Please check if it's showing properly in Galary once sample is pushed in GitHub.

Reply:
sure